RUFAS/routines/field/crop/nitrogen_incorporation.py

```diff
@@ -481,24 +497,61 @@ def _determine_shape_log(
             or emergence_nitrogen_fraction < 0
             or emergence_nitrogen_fraction > 1
         ):
+            om.add_error(
+                "Received invalid fractional value",
+                f"All following values must be in the range (0, 1), received {nitrogen_fraction=}, {heat_fraction=}"
+                f", {mature_nitrogen_fraction=}, {emergence_nitrogen_fraction=}.",
+                info_map,
+            )
             frac_error_msg = (
                 "nitrogen_fraction, heat_fraction, mature_nitrogen_fraction, and"
                 + " emergence_nitrogen_fraction must all be between 0 and 1"
             )
             raise ValueError(frac_error_msg)
-        if emergence_nitrogen_fraction == mature_nitrogen_fraction:  # leads to divide by zero
+        if emergence_nitrogen_fraction == mature_nitrogen_fraction:
+            om.add_error(
+                "Invalid value pair for emergence_nitrogen_fraction and mature_nitrogen_fraction.",
+                f"Equal emergence_nitrogen_fraction and mature_nitrogen_fraction will lead to zero"
+                f" division error,"
+                f"emergence_nitrogen_fraction and mature_nitrogen_fraction are both."
+                f" {emergence_nitrogen_fraction}",
```

Comment:
I liked the wording better on the half mature and mature heat fraction error messages and would suggest using that same format here.
```suggestion
                "A crop's emergence_nitrogen_fraction and mature_nitrogen_fraction are equal.",
                f"The emergence_nitrogen_fraction and mature_nitrogen_fraction are both"
                f" {emergence_nitrogen_fraction}, this results in a divide by zero error.",
```